tests/hpc/slurm_master.pm

```diff
@@ -420,20 +421,49 @@ sub extended_hpc_tests ($master_ip, $slave_ip) {
     parse_extra_log('XUnit', './results/TEST-hpc-test.xml');
 }
 
+our $slurm_pkg = 'slurm';
+
 sub run ($self) {
     select_serial_terminal();
     my $nodes = get_required_var('CLUSTER_NODES');
     my $slurm_conf = get_required_var('SLURM_CONF');
     my $version = get_required_var('VERSION');
 
+    if (get_var('SLURM_VERSIONED_TEST', '0') == '1') {
+        if (is_sle('<15-SP4')) {
+            record_info 'No testing', 'versioned Slurm not used';
+            return;
+        }
+        my $query_output = script_output(qq{zypper se slurm | awk '{print \$2}' | grep -E '^slurm_[0-9]{1,2}_[0-9]{1,2}\$' | uniq});
+        my @slurm_vers = split(/(\ |\n)/, $query_output);
+        my $higher_slurm_ver = 0;
+        record_info "@slurm_vers", 'slurm packages found';
+        foreach my $v (@slurm_vers) {
+            my ($slurm_version_tmp) = $v =~ /([0-9]{1,2}_[0-9]{1,2})/;
+            $slurm_version_tmp =~ s/_/\./;
+            if ($higher_slurm_ver < $slurm_version_tmp) {
+                $higher_slurm_ver = $slurm_version_tmp;
+            }
+        }
+        $higher_slurm_ver =~ s/\./_/;    # restore format
+        record_info "Latest $higher_slurm_ver", "This should be the latest/higher slurm version";
```

Comment:
Does it mean that there can be multiple versioned slurms? If yes, will all be supported ?  If so, we should test each version in defined job and not search use latest one. I would use `SLURM_VERSION` variable where we can define which version to test.

Reply:
This would complicate things. There was two versions. the lower one is the one which is installed by the non-version as far as i tell

Reply:
this also can resolved. With the second commit this block has been removed.